src/machma/bot.py

```diff
@@ -1,21 +1,24 @@
-from aiogram import Bot, Dispatcher, executor, types
+
+import html
 import logging
-import pprint
-from tabulate import tabulate
 import textwrap
-import html
 
-from .db import BotDB
+from aiogram import Bot, executor, types
+from tabulate import tabulate
+
+from . import api, db
+from .utils.aiogram.dispatcher import ProxyDispatcher
 
-API_TOKEN = open('./api_token.txt', 'r').read()
+dp = ProxyDispatcher()
 
-logging.basicConfig(level=logging.INFO)
 
-bot = Bot(token=API_TOKEN)
-dp = Dispatcher(bot)
-bot_db = BotDB('./bot.db')
+def run(api_token: str) -> None:
+    bot = Bot(token=api_token)
+    executor.start_polling(dp.to_dispatcher(bot), skip_updates=True)
+
 
 @dp.message_handler(commands=['help', 'hilfe', 'commands', 'befehle'])
+@db.async_session
```

Comment:
How does Telegram know what commands are available after replacing the annotation?

Reply:
You pass the commands to `message_handler()` right? :)

Reply:
Ah, of course I somehow thought that the annotation was replaced.